infra/README.md

```diff
@@ -0,0 +1,177 @@
+
+## Terraform Backend
+
+### infra/bootstrap/account
+
+1. Rename bootstrap/account, account directory to the name of the aws account alias or account id where this infrastructure will be hosted.
+2. Customize the variables in locals{} at the top of main.tf to match the desired deployment setup.
+3. Open a terminal and cd into the infra/bootstrap/account directory and run the following commands:
+    - terraform init
+    - terraform plan
+    - terrafrom apply
+4. Uncomment out the backend "s3" {} block, fill in the appropriate information from outputs and repeate step `3.` to switch from local to remote backend.
+
+``` tf
+  backend "s3" {
+    bucket         = "AWS_ACCOUNT_ID-AWS_REGION-tf-state"
+    key            = "terraform/backend/terraform.tfstate"
+    region         = "REGION_OF_BUCKET"
+    encrypt        = "true"
+    dynamodb_table = "tf_state_locks"
+  }
+```
+5. Once these steps are complete, this should not need to be touched again, application infrastructure is managed under its envs/environment as described below.
+
+Note: For subsequent accounts if using a multi-account setup, copy the entire account directory and repeat the previous steps for each account.
+
+### infra/envs/environment
+
+&nbsp;&nbsp;To get started with an environment copy the backend configuration created in the above instructions and copy into the terraform { backend "s3" {} } block to setup the remote backend for the environment. This is where all of the infrastructure for the application will be managed. 
+
+### Multi vs Single Cloud Account.
+
+&nbsp;&nbsp;In a multi-cloud account, multi-environment setup the relationship between the bootstrap/account(s) and envs/environement(s) should be 1:1. In a single-cloud account, multi-environment setup ensure that the backend "s3" { key = path/to/terraform.tfstate} is unique for the backend, as well as each environment.
+
+# Diagrams
+
+## Initial Setup
+<img src="../docs/imgs/initial_setup.svg" width="50%"/>
+
+## Multi-Cloud
+<img src="../docs/imgs/multi_cloud.svg" width="50%"/>
+
+## Single-Cloud
+<img src="../docs/imgs/single_cloud.svg" width="50%"/>
+
+
+# Workspaces
+
+### Terraform workspace commands:
+
+`terraform workspace show [Name]`   - This command will show the workspace you working in.
+
+`terraform workspace list [Name]`   - This command will list all workspaces.
+
+`terraform workspace new [Name]`    - This command will create a new workspace.
+
+`terraform workspace select [Name]` - This command will switch your workspace to the workspace you select.
+
+`terraform workspace delete [Name]` - This command will delete the specified workspace. (does not delete infrastructure, that step will done first)
+
+## Workspaces - Staging Environment
+&nbsp;&nbsp; Workspaces can be used here to allow multiple developers to deploy their own stacks for development and testing. If workspaces wont be necessary for this project set the prefix variable to "staging."
```

Comment:
```suggestion
## Template Repo Workspaces 
&nbsp;&nbsp; In this template repo, we've suggested that workspaces be used only for the staging environment.
```

Reply:
@shawnvanderjagt @aplybeah Do we want the environment that we set up for workspaces to be staging or test? I thought we said test